src/Randomizer.SMZ3/ILogic.cs

```diff
@@ -55,5 +55,11 @@ public interface ILogic
         public bool CanPassFireRodDarkRooms(Progression items);
 
         public bool CanParlorSpeedBoost(Progression items);
+
+        public bool CanPassMockBallRoom(Progression items);
+
+        public bool CanPassSwordOnlyDarkRoom(Progression items);
+
+        public bool CanHyruleSouthFakeFlippers(Progression items, bool fairyChests);
```

Comment:
Same as above. In this case I think it makes more sense to move `items.Flippers` out of this method, but if you have a better suggestion for the name to avoid that then that's fine too. I just want this to be clear at a glance when looking at some regions without having to check the definition of these methods.